src/vendorcode/dasharo/Kconfig

```diff
@@ -1,4 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
-config BIOS_VENDOR
```

Comment:
There were 2 BIOS_VENDOR definitions, this one seemed better to remove, because if the other one were removed, pre-commit was complaining that this one is missing some properties

Reply:
Yeah, this needs to be reverted.

Reply:
Reverted in 729902a0511ec6b28a2c7d7a22af4b005ba95975